homework/shelkovich_ilya/userValidationService/src/main/java/com/tutrit/java/quickstart/service/UserValidationService.java

```diff
@@ -0,0 +1,35 @@
+package com.tutrit.java.quickstart.service;
+
+import com.tutrit.java.quickstart.bean.User;
+import com.tutrit.java.quickstart.exceptions.UserValidationException;
+
+public class UserValidationService {
+
+    public boolean validate(User user) throws UserValidationException {
+        try {
+            checkUserNameLength(user);
+        } catch (UserValidationException nameException) {
+            return false;
+        }
+        try {
```

Comment:
I think you can use tone block "try" here, if you nedd it.

Reply:
fixed